pep-0008.txt

```diff
@@ -829,6 +829,15 @@ Note that there is a separate convention for builtin names: most builtin
 names are single words (or two words run together), with the CapWords
 convention used only for exception names and builtin constants.
 
+Type variable names
+~~~~~~~~~~~~~~~~~~~
+
+Type variable names should normally use CapWords preferring short
```

Comment:
I think this needs a quick link to PEP 484 since type variables are not a common thing in Python code other than code meant to be type-checked by mypy.

I would also like to see a few complete examples, e.g.
```
VT_co = TypeVar('VT_co', covariant=True)
```